examples/privileged-pod-pvc/README.md

```diff
@@ -0,0 +1,83 @@
+# Mount Volumes on Privileged Pods
+
+**This guide will demo GlusterFS as its example use-case but this method will work for any compatable volume provider.**
+
+##Purpose
+
+This example gives a basic template for attaching a persistent storage volume to a pod. It provides an end to end setup that begins with the _cluster-admin_ making the persistent volume availble and a _basic-user_ requesting storage from a **privileged** pod.
+
+_If the pod is not run as privileged, skip Edit Privileged SCC section_
+
+###Assumptions:
+
+* OSE 3.x
+* NFS, GlusterFS, Ceph, or other compatable volume provider
+* A cluster-admin user.  For this guide, that user is called `admin`
+
+##Create a basic-user and User Project
+
+_**Note:**_ This section assumes there are not yet basic users.  If you have a basic user and that user has a project, skip this section.
+
+	$ oc login -u tom -p tom
+    
+   Where "tom" is an arbitrary user name and password. 
+   
+   Next, create the project as tom:
+   
+   	$ oc new-project <project_name> \
+    --description="<description>" \
+    --display-name="<display_name>"
+   
+   _At a minimum, only `<project_name>` is required._
+
+   Basic-users are bound to the project-admin role at project creation so there is no need to manually bind them.
+
+##Edit Privileged SCC
+
+The privileged scc must be added to the user (or to a group that the user is a member of) before they can run privileged pods.
+
+_**As admin**_
+    
+    $ oc edit scc privileged
+    
+Under `users:` add the basic-user:
+
+    users:
+    - tom
+
+
+##Make the Volume Available to Projects
+
+_**As admin:**_
+
+	$ oc create -f gluster-endpoints.yaml
+	$ oc create -f gluster-service.yaml
+	$ oc create -f gluster-pv.yaml
+
+_**As basic-user**_
+
+	$ oc create -f gluster-pvc.yaml
+    $ oc create -f gluster-priv-pod.yaml
+    
+##Check the mount point inside the pod
+
+Get the pod name, then `oc exec` into it:
+
+	$ oc get pods
+    $ oc -it exec <pod_name> bash
+	[root@glusterfs-nginx-pvc /]# mount | grep -i gluster
+    
+Confirm that grep returns a line resemebling:
+
+	192.168.59.102:gv0 on /mnt/gluster type fuse.glusterfs (rw,relatime,user_id=0,group_id=0,default_permissions,allow_other,max_read=131072)
+
+
+**That's it!**
+
+##Relevent Origin Docs
+
+For more info on:
+
+* Setting pv/pvc's for other volume providers see [Configuring Persistent Storage](https://docs.openshift.org/latest/install_config/persistent_storage/index.html)
+* SCC's, see [Manageing Security Context Contraints]()
+
```

Comment:
do you want to show that the pod is using the privileged SCC that you edited above?